### ISSUE TITLE
Way of showing if user has not read editted message

### ISSUE THREAD

**Issue by ghost** [nathan-osman/auchatstatus#22]
If a user has read my message, and it is shown by where their little icon is, if I then edit that message I have no way of telling if they have read the edited version or not, so there should be some way of telling.

**Comment by ByteCommander** [comment]
Nice idea.

**Comment by nathan-osman** [comment]
How would this work?

**Comment by ghost** [comment]
I have been thinking about this too, and I was thinking about perhaps having the user icons of the users who have not read the edited version of the post on the post (as they would be if it was the last that they had read), except their icons would have a crack down the middle and would be in two pieces, and of course would have explanatory text in the tooltip.

**Comment by ByteCommander** [comment]
@SeventhSonOfASeventhSon That sounds too confusing.  
I'd rather let the script add additional post-like lines into the chat view that read e.g. like:

```
User [insert USERNAME and link to profile here] has edited their ["message", link to message] for the 1st/2nd/... time.
```

These lines should appear for own edits as well as for foreign edits as if they were posts created at the time the edit was made. Icons will be able to be located on them just like on normal posts.

How do you like this proposal?

**Comment by ghost** [comment]
I think it should not show who has read the new edit as then it may get confusing who hasn't, and it may also get a bit full. I think it should just have some way of showing who has read beyond that point but not seen the new edit.

**Comment by ByteCommander** [comment]
That's exactly what it does! You're confused and confusing... Do at least you understand what I mean @nathan-osman ?

**Comment by ghost** [comment]
So I'm like Fabby then! :D

**Comment by nathan-osman** [comment]
@ByteCommander yup, I get what you're saying but this could really clutter things up when a lot of people edit messages. Either way, this would be a low-priority feature due to its complex nature.

**Comment by ghost** [comment]
I think it would have to be something very minimal and non-cluttering, like what I suggested, or something similar, but not something that writes too much.

**Comment by ByteCommander** [comment]
And what about if you add a tool-tip when the mouse hovers over the edit symbol (pencil icon on the left of a message) that says one of the following options:
- Users x, y and z have seen this edit.
- Users x, y and z have not seen this edit yet.
- Users x, y and z have seen the original message, but not yet this edit.

I would go for the third one and only display the names of those who have read past this message already, but were not on the site after the edit was made yet. It should not show anybody's name who has not even read the original message, because that would be too much, I think. When all script users have noticed the edit (or not even seen the message at all), the tool-tip should vanish.

**Comment by ghost** [comment]
However this will work, it should definitely be able to show whether or not a user has seen the "(removed)" message only, or whether they have actually seen the original message before it was removed.
